docs/local.md

```diff
@@ -31,97 +42,145 @@ make build
 
 This creates `swapd` and `swapcli` binaries in the `bin` directory at the top of the project.
 
-To run as Alice, execute in terminal 1:
+### Launch Alice and Bob's swapd Instances
+
+To launch Alice's swapd instance, use this command:
 ```bash
-./swapd --dev-xmrtaker --deploy
+./bin/swapd --dev-xmrtaker --deploy &> alice.log &
```

Comment:
do we need to redirect to a log file for just local instructions? I think if people want to that they can do it on their own

Reply:
I was trying to simplify the instructions about using multiple terminals. Ideally, people are only looking at the logs to troubleshoot problems. I was assuming the same thing.  The user is free to not redirect the logs if they want them on the terminal.  They'd just cut-and-paste a short part of the line.